lineman/app/components/thread_page/comment_form/comment_form.coffee

```diff
@@ -11,8 +11,11 @@ angular.module('loomioApp').directive 'commentForm', ->
       $scope.isDisabled = true
       $scope.comment.save().then ->
         $scope.isDisabled = false
+        if $scope.comment.isReply()
+          FlashService.success('comment_form.messages.replied', name: $scope.comment.parentAuthorName())
```

Comment:
So James is saying that you need to remove the parentAuthorName() method you've added to commentModel.. because there is already a comment.parentAuthorName (not a function, just a property) available which does what you want and won't break.